core/src/main/java/com/opensymphony/xwork2/interceptor/ChainingInterceptor.java

```diff
@@ -160,7 +160,20 @@ private void copyStack(ActionInvocation invocation, CompoundRoot root) {
         Map<String, Object> ctxMap = invocation.getInvocationContext().getContextMap();
         for (Object object : list) {
             if (shouldCopy(object)) {
-                reflectionProvider.copy(object, invocation.getAction(), ctxMap, prepareExcludes(), includes);
+                String actionConfigClassName = invocation.getProxy().getConfig().getClassName();
```

Comment:
@lukaszlenart , I study a lot but, while user can equip any custom `ObjectFactory` to create actions (here user has many options like spring aop, javassist, jdk, cglib or any other library), it seems there is no way to determine which properties to set and which should be skipped **except asking from user itself**.

While there are `includes` and `excludes` available for user, but for save user from a lot of typing and also more readability, **what do you think about featuring a new optional parameter named `editableClass` like below?**
```xml
<result type="chain">
    <param name="actionName">myOriginalAction</param>
    <param name="editableClass">me.myname.myactions.MyOriginalAction</param>
</result>
```
or
```xml
<interceptor-ref name="chainStack">
    <param name="chain.editableClass">me.myname.myactions.MyOriginalAction</param>
</interceptor-ref>
```
After this featuring, rest of codes of this PR will work and fixes issue in a clean gracefully way.